packages/rln/src/index.ts

```diff
@@ -23,8 +19,7 @@ export {
   RLNDecoder,
   MerkleRootTracker,
   RLNContract,
-  RLN_STORAGE_ABI,
-  RLN_REGISTRY_ABI,
   SEPOLIA_CONTRACT,
-  extractMetaMaskSigner
+  extractMetaMaskSigner,
+  RLN_V2_ABI
```

Comment:
wdyt if we have just `RLN_ABI`? 
@adklempner @danisharora099 

my concern that `v1` is not supposed to function alongside `v2` so it's either or, so consumers (future) won't need to change variable names on each update

Reply:
can go either ways - nbd imo

Reply:
let's do just `rln` without version mention then